PetAdoption/src/main/java/dev/mongoose/models/PetPic.java

```diff
@@ -0,0 +1,63 @@
+package dev.mongoose.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * One foreign key to pet; A pet should be able to be created without a picture.
+ * A pet's picture can be updated and changed later.
+ * 
+ * @author Jacen
+ *
+ */
+@Entity
+@Table(name = "petpics")
+public class PetPic {
+
+    @Id
```

Comment:
PetId is a foreign key, if you use it as a primary key we will only be able to have one pic per pet